client/rest-high-level/src/test/java/org/elasticsearch/client/IndicesClientIT.java

```diff
@@ -183,6 +185,66 @@ public void testCreateIndex() throws IOException {
         }
     }
 
+    public void testGetSettings() throws IOException {
+        String indexName = "get_settings_index";
+        Settings basicSettings = Settings.builder()
+            .put("number_of_shards", 13)
```

Comment:
nit: could we create it with a bit less shards? I suspect it may make the test faster.